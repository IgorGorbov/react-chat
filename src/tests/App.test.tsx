// import * as React from 'react';
// import * as ReactDOM from 'react-dom';
// import toJSON from 'enzyme-to-json';
// import { shallow } from 'enzyme';
// import App from '../App';
//
// describe('testing App component', () => {
//   it('should contain Root component', () => {
//     const result = shallow(<App />);
//     expect(result.find('Chat')).toHaveLength(1);
//   });
//
//   it('renders without crashing', () => {
//     const div = document.createElement('div');
//     ReactDOM.render(<App />, div);
//     ReactDOM.unmountComponentAtNode(div);
//   });
//
//   it('snapshot App component', () => {
//     const wrapper = shallow(<App />);
//     const tree = toJSON(wrapper);
//     expect(tree).toMatchSnapshot();
//   });
// });
