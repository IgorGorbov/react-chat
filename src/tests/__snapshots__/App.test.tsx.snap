// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing App component snapshot App component 1`] = `<Chat />`;
